docs/user-guide.md

```diff
@@ -76,6 +76,15 @@ nix build -f . haskell-nix.nix-tools --out-link nt
 The easiest way to get a hold of [Haskell.nix][] is with
 [`fetchTarball`](https://nixos.org/nix/manual/#ssec-builtins).
 
+```nix
+import (builtins.fetchTarball https://github.com/input-output-hk/haskell.nix/archive/master.tar.gz + "/nixpkgs")
+  { nixpkgs-pin = "release-19.03"; }
+```
+The `nixpkgs-pin` argument is optional and specifies which of the nixpkgs versions.
+The supported options are `release-18.09`, `release-19.03` and `release-19.09` (the default).
```

Comment:
Isn't 19.03 the default? ci.nix seems to use  `"release-19.03"`

Reply:
19.09 is the one built and cached on hydra.iohk.io (though 19.03 is on there too, but less likely to be up to date).